Exiled.Events/Patches/Events/Player/ActivatingWorkstation.cs

```diff
@@ -0,0 +1,65 @@
+// -----------------------------------------------------------------------
+// <copyright file="ActivatingWorkstation.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Player
+{
+#pragma warning disable SA1313
+    using System;
+
+    using Exiled.Events.EventArgs;
+    using Exiled.Events.Handlers;
+
+    using HarmonyLib;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// Patch the <see cref="WorkStation.ConnectTablet(UnityEngine.GameObject)"/>.
+    /// Adds the <see cref="Player.ActivatingWorkstation"/> event.
+    /// </summary>
+    [HarmonyPatch(typeof(WorkStation), nameof(WorkStation.ConnectTablet))]
+    internal static class ActivatingWorkstation
+    {
+        private static bool Prefix(WorkStation __instance, GameObject tabletOwner)
+        {
+            try
+            {
+                if (!__instance.CanPlace(tabletOwner) || __instance._animationCooldown > 0f)
+                {
+                    return false;
+                }
+
+                global::Inventory component = tabletOwner.GetComponent<global::Inventory>();
```

Comment:
While not a problem, global:: declarations aren't necessary in C#, and usage of it doesn't really match the code style of the rest of the project.

This will not contribute to rejection of the PR, just merely an informational comment that you don't need to define global::

Reply:
Oops, didn't notice this when deleting dnspy ​​stuff.

Reply:
Accidentally copied it it from dnspy, fixed in 2fc1f4f